index.html

```diff
@@ -4,7 +4,67 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Kevin Weyrowitz</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <ul>
+        <li><a href="#Contact">Contact</a></li>
+        <li><a href="#Biography">Biography</a></li>
+      </ul>
+    </header>
+    <main>
+      <h1>Kevin Weyrowitz</h1>
+      <article>
```

Comment:
You could use the `<figure>` element with a nested `<figcaption>` element for this exact purpose.

Reply:
thx for your help :)